ocfweb/api/shorturls.py

```diff
@@ -0,0 +1,15 @@
+from django.http import HttpResponseNotFound
+from django.http import HttpResponseRedirect
+from ocflib.misc.shorturls import get_connection
+from ocflib.misc.shorturls import get_shorturl
+
+
+def bounce_shorturl(request, slug):
+    if slug:
+        with get_connection() as ctx:
+            target = get_shorturl(ctx, slug)
```

Comment:
Does this cause an exception/internal server error if the slug doesn't exist?

Reply:
no, the ocflib function returns `None`